### PR TITLE
Wip/auth demo improvements

### PR DESCRIPTION
**Pull request by ihor-il**
- Fixed password reset email text
- Used password reset tokens are now deleted from the database
- Password reset request now returns correct error messages
- Updated Log In notifications
- Added global toastr config (the same as in NotificationService)
- Added global error handler

### CODE REVIEW

File: frontend/src/app/app.module.ts [hunk 1]
```diff
@@ -42,10 +43,9 @@ import { UserModule } from './user/user.module';
     FormsModule,
     MatDialogModule,
     BrowserAnimationsModule,
-    ToastrModule.forRoot(),
+    ToastrModule.forRoot(ToatsrConfig),
```
Comment (ismv) [inline frontend/src/app/app.module.ts]: fix typo please

File: frontend/src/core/interceptors/global-error-handler.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  constructor(@Inject(Injector) private injector: Injector) {
+    super();
+  }
+
+  private get toastrService(): ToastrService {
+    return this.injector.get(ToastrService);
+  }
+
+  handleError(error: any): void {
+    if (
```
Comment (ismv) [inline frontend/src/core/interceptors/global-error-handler.ts:16]: github actions fix

Comment (ihor-il) [reply frontend/src/core/interceptors/global-error-handler.ts:16]: ErrorHandler uses any as parameter

File: frontend/src/app/app.module.ts (hunk 1 above)
Comment (ihor-il) [reply frontend/src/app/app.module.ts]: Done

**Review by Anatolii-Ivanov — APPROVED** [review]

**Review by ismv — APPROVED** [review]

**Pull request merged**